clients/da-vinci-client/src/main/java/com/linkedin/davinci/config/VeniceServerConfig.java

```diff
@@ -405,6 +409,8 @@ public VeniceServerConfig(VeniceProperties serverProperties, Map<String, Map<Str
       throws ConfigurationException {
     super(serverProperties, kafkaClusterMap);
     listenerPort = serverProperties.getInt(LISTENER_PORT, 0);
+    grpcPort = serverProperties.getInt(GRPC_READ_SERVER_PORT, 0);
+    isGrpcEnabled = serverProperties.getBoolean(ENABLE_GRPC_READ_SERVER, true);
```

Comment:
IMO default should be false
` serverProperties.getBoolean(ENABLE_GRPC_READ_SERVER, false);`